### PR TITLE
main/haproxy: upgrade to 1.9.7 and fix license

### PR DESCRIPTION
**Pull request by andypost**
Ref https://www.mail-archive.com/haproxy@formilux.org/msg33410.html

http://www.haproxy.org/download/1.9/src/CHANGELOG

### CODE REVIEW

File: main/haproxy/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributor: Jeff Bilyk <jbilyk@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=haproxy
-pkgver=1.9.6
+pkgver=1.9.7
 _pkgmajorver=${pkgver%.*}
 pkgrel=0
 pkgdesc="A TCP/HTTP reverse proxy for high availability environments"
```
Comment (maxice8) [inline main/haproxy/APKBUILD]: license is 'GPL-2.0-or-later  LGPL-2.1-or-later'

Comment (maxice8) [reply main/haproxy/APKBUILD]: empty variables can be removed for brevity

Comment (andypost) [reply main/haproxy/APKBUILD]: License changed, according to http://www.haproxy.org/download/1.5/doc/LICENSE

**Comment by maxice8** [comment]
Shouldn't it also have a $pkgname-openrc package because it provides an openrc service ?

File: main/haproxy/APKBUILD [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 url="http://haproxy.1wt.eu"
```
Comment (arambau) [inline main/haproxy/APKBUILD]: The URL for HAProxy should be https://www.haproxy.org/.

**Pull request closed without merging**